HARK/abstract/tests/consindshk.yml

```diff
@@ -0,0 +1,39 @@
+---
+states: !StateSpace
+  variables:
+    - !State
+      name: m
+      short_name: money
+      long_name: market resources
+      latex_repr: \mNrm
+    - !State
+      name: &name stigma
+      short_name: &short_name risky share
+      long_name: &long_name risky share of portfolio
+      latex_repr: &latex_repr \stigma
+
+actions: !ActionSpace
+  variables:
+    - !Action
+      name: c
+      short_name: consumption
+      long_name: consumption
```

Comment:
Are these fields optional? Can one spill over to the others as a default?
Basically, how can we make these config files lighter weight?

Reply:
These are optional, only required is name.